### DISCUSSION TITLE
Jenkins-Github-PR-Smee

### DISCUSSION THREAD

**Discussion by herczegzoltan** [bregman-arie/devops-exercises#263] [Q&A]
Hello,

I would have a question. currently we have setup a pipeline, so when there is a PR then on Jenkins a build is started. I used my personal account to connect Jenkins and GitHub (via personal access token), also using Smee for webhooks behind a secure firewall. My problem I would like to disconnect from my personal account but the organization does not provide any dummy email account to be used such as services@xxx.com.

How could I manage to be able to use PullRequestBuilder without any personal/dummy account?

Thank you

**Comment by Merajm07** [comment]
Yes , u can deactivate your account and again create a new account with dummy email 📨
